### PR TITLE
New PR to replace existing for Aggregations.io

### PR DESCRIPTION
**Pull request by cmastr**
This PR replaces this fork:

https://github.com/segmentio/segment-docs/pull/5887/files

### CODE REVIEW

**Review by tcgilbert — APPROVED** [review]

File: src/connections/destinations/catalog/actions-aggregations-io/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Aggregations.io (Actions) Destination
+id: 659eb601f8f615dac18db564
+beta: true
+---
+
+{% include content/plan-grid.md name="actions" %}
+
+[Aggregations.io](https://aggregations.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} enables you to utilize your existing analytics events & pipeline for real-time monitoring & alerting.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-aggregations-io/index.md:9]:
```suggestion
[Aggregations.io](https://aggregations.io/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} enables you to use your existing analytics events and pipeline for real-time monitoring and alerting.
```

File: src/connections/destinations/catalog/actions-aggregations-io/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This destination is maintained by Aggregations.io. For any issues with the destination, [contact their Support team](mailto:help@aggregations.io).
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure Aggregations.io (Actions)**.
+4. Select an existing Source to connect to Aggregations.io (Actions).
+5. In the destination settings, enter your Aggregations.io API Key & Ingest Id. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects` and the API Key requires `Write` permission.  For more information, see the [Aggregation.io Docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-aggregations-io/index.md:19]:
```suggestion
5. In the destination settings, enter your Aggregations.io API Key and Ingest ID. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects`, and the API Key requires `Write` permission.  For more information, see the [Aggregation.io docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.
```

**Review by rchinn-segment — CHANGES_REQUESTED** [review]
Just a few minor style updates

File: src/connections/destinations/catalog/actions-aggregations-io/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
```
Comment (rchinn-segment) [inline src/connections/destinations/catalog/actions-aggregations-io/index.md:19]:
```suggestion
5. In the destination settings, enter your Aggregations.io API Key and Ingest ID. Your ingestion on the Aggregations.io dashboard should be set up using `Array of JSON Objects` and the API Key requires `Write` permission.  For more information, see the [Aggregation.io docs](https://aggregations.io/docs/ingesting-data/create-an-ingest){:target="_blank"}.
```

**Review by rchinn-segment — APPROVED** [review]

**Pull request merged**